### PR TITLE
update by 14th of May

### PR DESCRIPTION
**Pull request by Drsheppard01**

### CODE REVIEW

File: pkgs.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "name": "acreom",
         "version": "1.5.8",
```
Comment (blueOkiris) [inline pkgs.json:9]: Forgot to version bump here. Should be 1.6.4

File: pkgs.json [hunk 2]
```diff
@@ -56,59 +61,74 @@
...
+        "url": "https://github.com/ArmCord/ArmCord/releases/download/v3.2.0/ArmCord-3.2.0.AppImage"
     }, {
         "name": "astrofox",
         "version": "1.4.0",
         "description": "Astrofox is a motion graphics program that lets you turn audio into amazing videos.",
         "url": "https://files.astrofox.io/download/Astrofox-1.4.0.AppImage"
     }, {
         "name": "audacity",
-        "version": "3.2.5",
+        "version": "3.3.0",
```
Comment (blueOkiris) [inline pkgs.json:74]: Should say 3.3.2 I believe

**Review by blueOkiris — CHANGES_REQUESTED** [review]
Needs just a couple fixes I think

**Review by blueOkiris — APPROVED** [review]

**Comment by Drsheppard01** [comment]
> Hey, FYI I just updated aip-man and added the feature to install from local repos.
> You might find this useful when doing these updates as you can test changes and things now before ever posting.
> To install from a different repo simply do `aipman -r <custom repo> install <package>`
> You can do this locally to test stuff by prepending the file path to the repo.json with `file://`

I had previously thought of this when using the NX Software Center in Nitrux (Linux distro that emphasizes AppImage). This is a very cool feature

I'm also currently working on an AppImage database something like an alternative to appimage.github.io or appimagehub.com consisting of aip-man pkg list

**Pull request merged**